#### Claire Darnell McWhite
##### claire.mcwhite@utexas.edu | clairemcwhite.github.io
##### Ph.D Candidate, Year 5, Cell and Molecular Biology Ph.D. Program, University of Texas at Austin
 
**Education**

* University of Texas at Austin, Cell and Molecular Biology Program, Computational Biology (2014-)
* Rice University, Class of 2014, B.S. Biochemistry and Cell Biology 
 
**Research Experience**

*The University of Texas at Austin Department of Molecular Biosciences*

* Dr. Edward Marcotte Research Group (2014-) Projects relating to computational prediction of protein complexes, including 1) conserved complexes of plants and 2) stable complexes of the cilia, and comparative evolution of proteins across eukaryotes.
* Dr. Claus Wilke Research Group (2/2015-8/2015) Projects relating to uncovering sources of artifacts in computational evolutionary rate measurement of influenza evolution.
* Dr. John Wallingford Research Group (11/2014-2/2015) Projects relating to functionally replacing genes in higher organisms with genes from single celled organisms.

*Rice University Department of Chemical and Biomolecular Engineering*
             
* (2010-2014) Dr. Laura Segatori Research Group, Cell and Protein Engineering Laboratory. Projects relating to the discovery of novel proteasome activating compounds and genes in mammalian and yeast models.  

*National Cancer Institute, National Institutes of Health, Bethesda, MD*

* (2008-2012) Dr. Ilona Linnoila Research Group, Experimental Pathology Section, Repeated internships. Projects related to lung epithelial cells in abnormal lesions and chemical injury.
 
**Awards**

* NIH F31 Graduate Fellowship, ‘Proteomics of ciliopathy protein complexes’ – Impact score: 10, Percentile: 1* , 1/2018
* Graduate School Summer Fellowship, Summer, 2017
* Honorable Mention, NSF Graduate Research Fellowship Program (2016)
* Spring/Fall Graduate Fellowship, UT Austin (8/2014-5/2015)
* National Merit Scholarship (8/2010-5/2014)
* Trustee's Distinguished Scholarship, Rice University (8/2010-5/2014)
* Century Scholarship for Research, Rice University (8/2010-5/2012)
 
**Honors**

* Best Talk, First International Plant Systems Biology Conference, Fall, 2018
* Student Choice Award, UT Austin Natural Sciences Council’s Art in Science Display, 3/2017, for my network visualization of gene orthology relationships.
* Visualizing Science Competition, 2nd Place, UT Austin College of Natural Sciences, 11/2016, my visualization of gene-gene relationships is hanging in the departmental office.
* Graduate Student Research Award, Society of Systemic Biologists, 7/2016
* Selected Artist, Evolution Art Exhibition, Art.Science.Gallery, 6/2016
* Best Poster, UT Austin Institute for Cellular and Molecular Biology Retreat, 9/2015
* Best Poster, Big Data in Biology Symposium, Center for Computational Biology and Bioinformatics, Austin, Texas, 5/2015
 
**Publications**

* W Zhao, B Bachhav, C McWhite, L Segatori, "A yeast selection system for the detection of proteasomal activation", Protein Engineering Design and Selection 2019:gzz006
* A Zeileis, JC Fisher, K Hornik, R Ihaka, CD McWhite, P Murrell, R Stauffer, CO Wilke, "colorspace: A Toolbox for Manipulating and Assessing Colors and Palettes", arXiv:1903.06490
* AH Kachroo, JM Laurent, A Akhmetov, M Szilagyi-Jones, CD McWhite, A Zhao, EM Marcotte. “Systematic bacterialization of yeast genes identifies a near-universally swappable pathway", eLife 2017;6:e25093
* KS Drew, C Lee, RL Huizar, F Tu, B Borgeson, CD McWhite, Y Ma, JB Wallingford, EM Marcotte, “Integration of over 9,000 mass spectrometry experiments builds a global map of human protein complexes”, Molecular Systems Biology (2017) 13, 932
* BJ Liebeskind, CD McWhite, EM Marcotte, “Towards Consensus Gene Ages”, Genome Biology and Evolution 8 (6): 1812-1823, 2016
* CD McWhite, AG Meyer, CO Wilke, “Sequence amplification via cell passaging creates spurious signals of positive adaptation in influenza virus H3N2 hemagglutinin”, Virus Evolution 2 (2):vew026, 2016
* CD McWhite, BJ Liebeskind, EM Marcotte, “Applications of comparative evolution to human disease genetics”, Current Opinion in Genetics & Development 35, 16-24, 2015 
* W Zhao, M Bonem, CD McWhite, JJ Silberg, L Segatori, “Sensitive detection of proteasomal activation using the Deg-On mammalian synthetic gene circuit”, Nature Communications 5, 2014
* A Demelash, P Rudrabhatla, HC Pant, X Wang, ND Amin, CD McWhite, X Naizhen, RI Linnoila, “Achaete-scute homologue-1 (ASH1) stimulates migration of lung cancer cells through Cdk5/p35 pathway”, Molecular Biology of the cell 23 (15), 2856-2866, 2012
 
**Outreach**

* Author of colorblindr: An R package to aid design of colorblind-interpretable figures
* Introduction to Network Visualization, Austin R Users Group, Spring, 2018
* Speaker at BAHFest Texas, A festival of bad ad-hoc hypotheses, Spring, 2018
 
**Talks**

* Computing on Phenotypes, Ames, IA, 4/2019
* First International Plant Systems Biology Conference, Roscoff, France, 9/2018

**Posters**

* The Society for Molecular Biology and Evolution, Austin, TX, 7/2017
* Omics Approaches to Study the Proteome Keystone Conference, Breckenridge, CO, 1/2017
* Plant Biology, Austin, TX, 7/2016
* Evolution, Austin, TX, 6/2016
* Quest for Orthologs Meeting, Barcelona, Spain, 5/2015
* Big Data in Biology Symposium, Austin, TX, 5/2015
* Rice Undergraduate Research Symposium, 4/2011, 4/2012, 4/2013
* NIH Summer Research Symposium, 8/2011
 
**Teaching**

* Teacher’s Assistant – Intro to Next-Generation Sequencing Short Course, Spring, 2018
* Co-instructor – Peer-led Biocomputing Working Group: Python/Bash, Spring, 2017
* Lecturer (data visualization in R) –Peer-led Biostatistics Seminar, Fall, 2016
* Open Coding Hour – Assist graduate students with programming issues. Fall 2016 – Spring, 2018
* Teacher’s Assistant – BCH339N Systems Biology & Bioinformatics, Spring, 2016
 
**References**

* Prof. Edward Marcotte – marcotte@icmb.utexas.edu
* Prof. John Wallingford – wallingford@austin.utexas.edu
* Prof. Claus Wilke – wilke@austin.utexas.edu
 
